examples/triton_gpt2/README.ipynb

```diff
@@ -129,7 +129,7 @@
    ],
    "source": [
     "!mc mb minio-seldon/onnx-gpt2 -p\n",
-    "!mc cp ./model.onnx minio-seldon/onnx-gpt2/gpt2/1/"
+    "!mc cp ./model.onnx minio-seldon/onnx-gpt2"
```

Comment:
Triton is very strict on the folder structure, are we sure that removing the subfolders will not break anything here?